xiblint/rules/__init__.py

```diff
@@ -4,14 +4,28 @@
 from importlib import import_module
 
 
+class Rule(object):
+    def __init__(self, config):
+        self.config = config
+
+    def check(self, context):
+        raise NotImplementedError()
+
+
+def _class_name_for_file(filename):
+    components = filename.split("_")
+    return "".join(x.title() for x in components)
```

Comment:
Alternative: Could also be written as

return filename.title().replace(“_”, “”)

Reply:
nice, done